lib/commands/kendoui-base.ts

```diff
@@ -0,0 +1,73 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import Future = require("fibers/future");
+import * as assert from "assert";
+import * as helpers from "../common/helpers";
+
+export class KendoUIBaseCommand implements ICommand {
+
+	constructor(protected $errors: IErrors,
+		protected $project: Project.IProject,
+		protected $kendoUIService: IKendoUIService,
+		protected $loginManager: ILoginManager,
+		protected $options: IOptions) {	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	canExecute(args: string[]): IFuture<boolean> {
+		return ((): boolean => {
+			if(args && args.length > 0) {
+				this.$errors.fail("This command does not accept parameters.");
+			}
+
+			if(this.$options.core && this.$options.professional) {
+				this.$errors.fail("You cannot specify core and professional options simultaneously.");
+			}
+
+			return true;
+		}).future<boolean>()();
+	}
+
+	execute(args: string[]): IFuture<void> {
+		assert.fail("","", "You should never get here. Please contact Telerik support and send the output of your command, executed with `--log trace`.");
```

Comment:
Well I picked it up from another set of hierarchical commands - [Build](https://github.com/Icenium/icenium-cli/blob/8423abb30339d8b6d4fcd1753034bf4d87e9e04f/lib/commands/build.ts#L14).

Ping @rosen-vladimirov @teobugslayer for opinions?